ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2075,6 +2076,33 @@ snmp_server:
       enable: < true | false >
 ```
 
+#### Monitor Sessions
+
+```yaml
+monitor_sessions:
+  - name: < session_name_1 >
+    sources:
+      - name: < interface_name or range >
+        direction: < rx | tx | both >
+        access_group:
+          type: < ip | ipv6 | mac >
+          name: < acl_name >
+          priority: < priority >
+    destinations:
+      - < interface(s) | cpu >
```

Comment:
I have no idea about this. So just wanted to know. Does this knob support any type of interfaces (vlan, lo) or only Ethernet ?

Reply:
commonly ethernet interfaces, portchannels or CPU

apparently some platforms also suppor to mirror to a gre tunnel:

https://www.arista.com/en/um-eos/eos-data-transfer#xx1135616